results/final_results.json

```diff
@@ -15,9 +15,9 @@
         "standard": 16.43666922426724
     },
     "htmcore": {
-        "reward_low_FN_rate": 62.70275034462445,
-        "reward_low_FP_rate": 46.55193119229405,
-        "standard": 57.22915150504096
+        "reward_low_FN_rate": 76.5626293570994,
+        "reward_low_FP_rate": 61.359926511549155,
+        "standard": 71.3094612770284
```

Comment:
please rerun for the new (worse :( ) scores. And you can update README with the new results! :clap: